references/cli/addon-registry.go

```diff
@@ -79,6 +79,22 @@ add a gitlab registry: vela addon registry add my-repo --type gitlab --endpoint=
 			if err != nil {
 				return err
 			}
+			if registry.Helm == nil {
+				_, err = registry.ListAddonMeta()
```

Comment:
maybe we don't check if the registry is not helm

Reply:
don't check if the registry is not helm